migrations/versions/2018_12_03_remove_duplicate_fkey.py

```diff
@@ -0,0 +1,34 @@
+"""Separate data sources from the page table
```

Comment:
Should be "Remove duplicate foreign key"?

Reply:
Sorted, good spot